README.md

```diff
@@ -25,7 +25,7 @@ To run a Renaissance benchmark, you need to have a JRE version 11 (or later)
 installed and execute the following `java` command:
 
 ```
-$ java -jar 'renaissance-gpl-0.15.0.jar' <benchmarks>
+$ java -jar 'renaissance-gpl-0.16.0.jar' <benchmarks>
```

Comment:
@lbulej, should the version bump happen in such a PR? I think we usually do not touch the version number for features/benchmarks, but only do when we actually do a release?

Reply:
It should not. I think this might be the result of not pulling tags from the remote when cloning the repository. In such case, I think the version from `version.sbt` will be used, which is always one step ahead of the current version. Ilia, if you  pull the tags to your local clone, rebuild renaissance and then run the archive with `--readme` command, it should generate an updated `README.md` (with the benchmark) but without the version change.